peripherals/example.yaml

```diff
@@ -29,6 +29,9 @@ info:
   version: 0.1.0
   'x-foo': 'x-bar'
 
+imports:
```

Comment:
It seems like this change may cause unit tests to fail as Emboss is currently not in the CI environment. If we revert this change and put together a separate `peripherals/example-emboss.yaml` file it should not cause any CI failures.